application/org.openjdk.jmc.flightrecorder.flameview/src/main/java/org/openjdk/jmc/flightrecorder/flameview/views/FlameGraphView.java

```diff
@@ -77,7 +77,13 @@
 import org.openjdk.jmc.ui.handlers.MCContextMenuManager;
 import org.openjdk.jmc.ui.misc.DisplayToolkit;
 
+import static org.openjdk.jmc.flightrecorder.stacktrace.messages.internal.Messages.STACKTRACE_UNCLASSIFIABLE_FRAME;
```

Comment:
Messages to be shared should not be in an internal bundle.

Reply:
I've put them into the move them into the package : org.openjdk.jmc.flightrecorder.stacktrace.messages.common.